driver/normalizer/annotation.go

```diff
@@ -87,5 +131,47 @@ var AnnotationRules = On(Any).Self(
 			On(HasProperty("Tok", "goto")).Roles(uast.Goto),
 			// TODO: fallthrough
 		),
+
+		// Switch statement
+		On(goast.SwitchStmt).Roles(uast.Switch),
+		On(goast.CaseClause).Roles(uast.Case),
+		// TODO: default (length of List is zero)
+
+		// Declarations
+		On(goast.GenDecl).Roles(uast.Declaration).Self(
+			On(HasProperty("Tok", "var")).Roles(uast.Variable).Children(
+				OnIntRole("Specs", uast.Variable).Children(
+					OnIntRole("Names", uast.Variable),
+				),
+			),
+			// TODO: constants
+			//On(HasProperty("Tok","const")).Roles(uast.Constant).Children(
+			//	OnIntRole("Specs", uast.Constant).Children(
+			//		OnIntRole("Names", uast.Constant),
+			//	),
+			//),
+		),
+
+		// Imports
+		On(goast.ImportSpec).Roles(uast.Import, uast.Declaration).Children(
```

Comment:
Is there info about the aliasing in the AST? (like when you do `import x "path_to_module"` with x being the alias)? In you can extract that there is a `uast.Alias` role that could be applied.

Reply:
Good catch! Missed it somehow in other drivers.

Reply:
Np, annotating is hard because of the "fuzzyness" of matching the UAST to the AST and you're doing it really well considering that it's your first driver.